src/luarocks/core/cfg.lua

```diff
@@ -508,6 +508,22 @@ local function make_rocks_provided(lua_version, luajit_version)
       rocks_provided_3_0["tarantool"] = tarantool_version.."-1"
    end
 
+   if cfg.lua_interpreter == "tt" then
+      -- Run from tt.
+      local output = util.popen_read("tarantool --version 2>/dev/null")
```

Comment:
What output we get in case the `tarantool` binary is missing?
In the case of `tt`, `tarantool` from `env` must be used. This must be taken into account when running luarocks from `tt`.

Reply:
//What output we get in case the tarantool binary is missing?
It will be empty.

Reply:
What env? Maybe call `tt run -e 'print(_TARANTOOL)'`